<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginPage</class>
 <widget class="QDialog" name="loginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formula Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgba(30,30,30,255);
	border-radius:10px;
}</string>
  </property>
  <widget class="QLabel" name="LoginLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>331</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-family: Open Sans Light;
	font-size: 25px;
	color: #FFF;
	
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	height: 30px;
	color: #FFF;
	border-radius: 18px;
	font-family: Open Sans Light;
	font-size: 20px;
	background-color: rgba(13,117,190,255);
    
}

QPushButton:hover {
	border-radius: 20px;
	background-color: rgb(85,150,190);
}

QPushButton:pressed {
	border-radius: 20px;
	background-color: rgb(28,81,115);
	
}</string>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernameEditLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans 61</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font-family: Open Sans Light;
	border: 2px solid rgb(37, 39, 48);
	border-radius: 16px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(34, 36, 44);
}

QLiteEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}

QLiteEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}
</string>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
   <property name="placeholderText">
    <string>Usu√°rio</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEditLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans 61</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font-family: Open Sans Light;
	border: 2px solid rgb(37, 39, 48);
	border-radius: 16px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(34, 36, 44);
}

QLiteEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}

QLiteEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}
</string>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Senha</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="MostrarPasswordButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>226</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: Open Sans Light;
	background: transparent;
	border: none;


}</string>
   </property>
   <property name="text">
    <string>Mostrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
